### PR TITLE
Get orchestration running again

### PR DESCRIPTION
**Pull request by Rizato**
Adds the now required postgres_password env var. 
Pins ipfs to a working version. (Must delete the ipfs volumes if a run was attempted)
Adds polyswarmd new env vars.

### CODE REVIEW

**Comment by Rizato** [comment]
This revealed a polyswarm-client bug that needs to be fixed as well.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -3,12 +3,19 @@ image: docker:stable
...
 
+variables:
+  DOCKER_HOST: tcp://localhost:2376/
+  DOCKER_TLS_CERTDIR: "/certs"
+  DOCKER_TLS_VERIFY: 1
+  DOCKER_CERT_PATH: "$DOCKER_TLS_CERTDIR/client"
+
 before_script:
+  - sleep 2
   - apk --no-cache add curl jq
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:14]: need to use `apt-get`

Comment (Rizato) [reply .gitlab-ci.yml:14]: I'm thinking about just remove the gitlab stuff from this project. I don't think it really does anything.

**Review by mrtizmoatwork — APPROVED** [review]
ok

**Pull request merged**